recipes/joommfutil/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.7" %}
+
+package:
+  name: joommfutil
+  version: {{ version }}
+
+source:
+  fn: joommfutil-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/81/0e/b50101da4fc63c7dab5cc97231141c1f41871d740805fbdcf216cc83dae7/joommfutil-0.7.tar.gz
+  sha256: a9c92c05b0c0c8af79de38e3f943dc09ca159f21c7ce6e8dcaa7027499be7a9d
+
+build:
+  number: 0
+  script: pip install --no-deps .
```

Comment:
I think the `pip` installation is preferable, and I've used this in a number of other recipes. See also discussion on #3034